packages/database/src/utilities/mapOverlays.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+export const getMapOverlayGroupId = async (db, code) => {
+  const results = await db.runSql(`select "id" from "map_overlay_group" where "code" = '${code}';`);
+
+  if (results.rows.length > 0) {
+    return results.rows[0].id;
+  }
+
+  throw new Error('MapOverlayGroup not found');
+};
+
+export const getPermissionGroupId = async (db, name) => {
```

Comment:
I think this exact function has been declared individually in about 20 different migrations, so probably a good call to split it out to a util function, but I wouldn't expect to find this in a file called `mapOverlays` 🤔 

I reckon get rid of the mapOverlay function for the generic codeToId and then rename this file to match this util function